src/vm/nativeeventsource.h

```diff
@@ -0,0 +1,22 @@
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+// File: nativeeventsource.h
+// Abstract: This module implements native part of Event Source support in VM
+//
+
+//
+
+//
+// ============================================================================
```

Comment:
Could you please the add the ifdef that allows this header file to be included multiple times - like we have in all other header files:

```
#ifndef _NATIVEEVENTSOURCE_H_
#define _NATIVEEVENTSOURCE_H_
```